deploy/crds/broker_v1alpha1_activemqartemis_crd.yaml

```diff
@@ -31,6 +31,11 @@ spec:
           properties:
             image:
               type: string
+            size:
+              type: integer
+              minimum: 1
+              maximum: 16
+              description: The number of activemqartemis pods to deploy
```

Comment:
This should probably also be specified as a required field. Otherwise, I believe it can be left out and would then default to 0, and from your code perspective, it will appear that it was specified with a value of zero. Otherwise and if you want it as optional (and pick your own default), you need to make it a pointer and check for nil and set it.

Reply:
Mark as required, I think this better way and the user will have the idea where to increase the number of replica's and if not mentioned stop and alert the user at the first step

Reply:
Then I suggest you also add it to any example CR you have, if not there, and most importantly, to the OLM example yaml.

Reply:
that's mentioned in CR and OLM example like here https://github.com/rh-messaging/activemq-artemis-operator/blob/50ddf151ce1f1e3870bac6f80d465c5d30b3e6c1/deploy/catalog_resources/redhat/activemq-artemis-operator.v0.4.0.clusterserviceversion.yaml#L26